test/test_common.h

```diff
@@ -7,4 +7,126 @@
 #ifndef __RPMI_TEST_COMMON__
 #define __RPMI_TEST_COMMON__
 
+#include <stdio.h>
+#include <librpmi.h>
+#include <stdlib.h>
+
+/*
+ * Qemu shared memory offsets.
+ * its better to decode these data from DTS file, hardcoding for now.
+ */
+
+#define RPMI_SHM_XPORTS       (2)
+/*
+ * test assums qemu RPMI shared memory address is mapped at the following
+ * offset in gues RAM
+ */
+#define RPMI_SHM_OFSET       (0x0000000010001000)
```

Comment:
We don't need lot of these defines.

Reply:
agreed, was needed for qemu, not anymore dropping.